src/structures/ClientUser.js

```diff
@@ -95,10 +95,10 @@ class ClientUser extends Structures.get('User') {
    * @typedef {Object} PresenceData
    * @property {PresenceStatusData} [status] Status of the user
    * @property {boolean} [afk] Whether the user is AFK
-   * @property {Object} [activity] Activity the user is playing
-   * @property {string} [activity.name] Name of the activity
-   * @property {ActivityType|number} [activity.type] Type of the activity
-   * @property {string} [activity.url] Twitch / YouTube stream URL
+   * @property {Object[]} [activities] Activity the user is playing
+   * @property {string} [activities[].name] Name of the activity
+   * @property {ActivityType|number} [activities[].type] Type of the activity
+   * @property {string} [activities[].url] Twitch / YouTube stream URL
```

Comment:
have you tested if this works with the docs? if not perhaps make a type for `activities`

Reply:
This is how it looks in the docs:
![1](https://user-images.githubusercontent.com/51517103/107861968-70edf400-6e6f-11eb-8c58-b59a7fbac4dd.png)

Normal jsdoc for comparison:
![2](https://user-images.githubusercontent.com/51517103/107861970-73504e00-6e6f-11eb-8e36-63c6c8ac27b3.png)

How about we make a new `ActivitiesOptions` type, with `name`, `type`, and `url` property. Then we can document this `PresenceData#activities` to be an array of `ActivitiesOptions`.

Reply:
yeah that would be best, because `activities[].name` looks kinda funky